src/makeCalculator.js

```diff
@@ -4,7 +4,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  return {
+    add(a, b) {
+      return a + b;
+    },
+    subtract(a, b) {
+      return a - b;
+    },
+    multiply(a, b) {
+      return a * b;
+    },
+    divide(a, b) {
+      if (b === 0) {
+        return 'Cannot divide by zero';
+      }
+
+      return a / b;
+    },
+    operate(operation, number) {
+      result = operation(result, number);
+
+      return this;
```

Comment:
[NAMING] - The `operate` method is correctly implemented to take a callback and a number and allows method chaining. However, ensure that the callback is a function to avoid runtime errors. Use `typeof operation === 'function'` to validate the callback.